include/openssl/ts.h

```diff
@@ -635,6 +662,7 @@ void ERR_load_TS_strings(void);
 # define TS_F_TS_RESP_CTX_SET_ACCURACY                    128
 # define TS_F_TS_RESP_CTX_SET_CERTS                       129
 # define TS_F_TS_RESP_CTX_SET_DEF_POLICY                  130
+# define TS_F_TS_RESP_CTX_SET_ESS_CERT_ID_V2_DIGEST       159
```

Comment:
Define TS_F_TS_RESP_CTX_SET_ESS_CERT_ID_V2_DIGEST is not used anywhere in the code.